src/OrbitGl/App.cpp

```diff
@@ -1167,42 +1168,41 @@ void OrbitApp::LoadSymbols(const std::filesystem::path& symbols_path, ModuleData
     scoped_status.UpdateMessage(message);
     LOG("%s", message);
 
-    main_thread_executor_->Schedule(
-        [this, scoped_status = std::move(scoped_status), module_data,
-         function_hashes_to_hook = std::move(function_hashes_to_hook),
-         frame_track_function_hashes = std::move(frame_track_function_hashes)] {
-          modules_currently_loading_.erase(module_data->file_path());
-
-          const ProcessData* selected_process = GetTargetProcess();
-          if (selected_process != nullptr &&
-              selected_process->IsModuleLoaded(module_data->file_path())) {
-            functions_data_view_->AddFunctions(module_data->GetFunctions());
-            LOG("Added loaded function symbols for module \"%s\" to the functions tab",
-                module_data->file_path());
-          }
+    main_thread_executor_->Schedule([this, scoped_status = std::move(scoped_status), module_data,
+                                     function_hashes_to_hook = std::move(function_hashes_to_hook),
+                                     frame_track_function_hashes =
+                                         std::move(frame_track_function_hashes)] {
+      modules_currently_loading_.erase(module_data->file_path());
+
+      const ProcessData* selected_process = GetTargetProcess();
+      if (selected_process != nullptr &&
+          selected_process->IsModuleLoaded(module_data->file_path())) {
+        functions_data_view_->AddFunctions(module_data->GetFunctions());
+        LOG("Added loaded function symbols for module \"%s\" to the functions tab",
+            module_data->file_path());
+      }
```

Comment:
I think some unrelated clang-format changes have slipped into this commit. Or am I not seeing the change here?

Reply:
Looks like it, this should not be here. I can revert this and push it in a separate PR since the changes seem to be reasonable, WDYT?

Reply:
My PR #1752 is going to change that anyway. So for me it's fine if you just remove the change. And I have to admit, my motivation to report this was mainly driven by the fact that this would lead to a merge conflict in my PR. :sweat_smile:

Reply:
Thank you. :innocent: